packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1759,6 +1785,43 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   AutofillClient get _effectiveAutofillClient => widget.autofillClient ?? this;
 
+  late SpellCheckConfiguration? _spellCheckConfiguration;
+
+  late SpellCheckSuggestionsHandler? _spellCheckSuggestionsHandler;
+
+  /// Configuration that determines how spell check will be performed.
+  ///
+  /// This configuration will attempt to provide a default for the
+  /// [SpellCheckService] if it is not specified.
+  ///
+  /// See also:
+  ///  * [DefaultSpellCheckService], the default spell check service.
+  @visibleForTesting
+  SpellCheckConfiguration? get spellCheckConfiguration => _spellCheckConfiguration;
+
+  /// The hander used to display spell check results.
+  ///
+  /// This is only initialized if spell check is enabled.
+  @visibleForTesting
+  SpellCheckSuggestionsHandler? get spellCheckSuggestionsHandler => _spellCheckSuggestionsHandler;
+
+  bool _spellCheckEnabled = false;
```

Comment:
Can this be derived from the state rather than saved?  For example, maybe this could be a getter that looks at widget.spellCheckConfiguration to determine its return value.  That way we don't have to worry about this value ever being out of sync with widget.spellCheckConfiguration.